### PR TITLE
Make show/hide toolbox button in p5 labs tab-navigable

### PR DESCRIPTION
**Pull request by molly-moen**
This button was previously a span, this PR changes it to a button. The only visual difference is there is a border around the button while you are clicking on it, otherwise it looks the same. As we've been updating this button and the settings cog the spacing has changed a little in the "show toolbox" mode. This PR, along with the one for the [settings cog](https://github.com/code-dot-org/code-dot-org/pull/49248) also improve the right to left language behavior of these buttons, previously they overlapped.

## Before
Left to Right language
Screen Shot 2022-12-06 at 9 59 30 AM
Screen Shot 2022-12-06 at 9 57 01 AM

Right to Left language
Screen Shot 2022-12-06 at 9 59 15 AM
Screen Shot 2022-12-06 at 9 57 27 AM

## After 
Left to Right language
Screen Shot 2022-12-06 at 9 54 41 AM
Screen Shot 2022-12-06 at 9 54 51 AM

Right to Left language
Screen Shot 2022-12-06 at 9 54 12 AM
Screen Shot 2022-12-06 at 10 01 27 AM

## Links

- jira ticket: [SL-335](https://codedotorg.atlassian.net/browse/SL-335) I am splitting this ticket into multiple PRs because the buttons are all unrelated and they each have slight risk of eyes diffs.

## Follow-up work
The left to right behavior is still not ideal for droplet, as the "show/hide toolbox" menu is on the right but the toolbox is on the left. This requires additional work because `CodeWorkspace` is used for both Blockly and Droplet labs, and the Blockly toolbox is on the right. A follow-up task to untangle this is [here](https://codedotorg.atlassian.net/browse/SL-428)

### CODE REVIEW

File: apps/src/templates/CodeWorkspace.jsx [hunk 1]
```diff
@@ -294,6 +289,23 @@ const styles = {
     padding: 5,
     opacity: 0.9,
     position: 'relative'
+  },
+  chevronButton: {
```
Comment (sanchitmalhotra126) [inline apps/src/templates/CodeWorkspace.jsx:299]: Out of curiosity, where was all this styling set previously?

**Review by sanchitmalhotra126 — APPROVED** [review]

**Review by fisher-alice — APPROVED** [review]
Great idea to split this ticket due to possible eyes diffs!
Fyi - I think the link to the ticket may need to be tweaked.

File: apps/src/templates/CodeWorkspace.jsx (hunk 1 above)
Comment (molly-moen) [reply apps/src/templates/CodeWorkspace.jsx:299]: Some of it was in `chevron`, above, which I renamed to make more clear. The rest is so a button does not have default button styling (such as a gray background and extra padding)

**Comment by bencodeorg** [comment]
@molly-moen do you mind if I merge this first? Think it'll be easier to update this PR than vice versa. I can fix the merge conflicts if you'd like

https://github.com/code-dot-org/code-dot-org/pull/49248

**Comment by molly-moen** [comment]
> @molly-moen do you mind if I merge this first? Think it'll be easier to update this PR than vice versa. I can fix the merge conflicts if you'd like
> 
> #49248

@bencodeorg go for it! I've got a build failure I have to figure out anyway. I can resolve diffs :)

**Comment by bencodeorg** [comment]
Great, thanks! Also have a build failure to sort out, but think it's transient. Will update here when it's merged.

**Comment by bencodeorg** [comment]
@molly-moen just merged this so there are now conflicts here, LMK if you need any help resolving:

https://github.com/code-dot-org/code-dot-org/pull/49248

**Pull request merged**